resources/views/createCourse.blade.php

```diff
@@ -0,0 +1,145 @@
+
+@extends('layouts.app')
+
+@section('title', 'Register a course')
+
+@section('content')
+<div class="padded content">
+	<h1>Register a course</h1>
+	<form class="ui error form" method="POST" action="/courses">
```

Comment:
hmmm are you actually performing that action? puedes ignorar esto si sí pero suena a que tú vas a mandar el request por ajax abajo?

Reply:
El form se manda como POST a "/courses" al hacer submit, no ajax request here :)